azure-iot-device/tests/iothub/aio/test_async_clients.py

```diff
@@ -1517,7 +1517,7 @@ class TestIoTHubModuleClientReceiveTwinDesiredPropertiesPatch(
 class TestIoTHubModuleClientInvokeMethod(IoTHubModuleClientTestsConfig):
     @pytest.mark.it("Begins a 'invoke_method' HTTPPipeline operation where the target is a device")
     async def test_calls_pipeline_invoke_method_for_device(self, mocker, client, http_pipeline):
-        method_params = "__fake_method_params__"
+        method_params = {"methodName": "__fake_method_name__"}
```

Comment:
>"methodName" [](start = 25, length = 12)

In the docstring we actually say it's "method_name". Not sure which is correct

Reply:
methodName is what the implementation has.  Changing it to method_name at this point would be a breaking change.  This is why we have [https://github.com/Azure/azure-iot-sdk-python/issues/633](https://github.com/Azure/azure-iot-sdk-python/issues/633) 

---
In reply to: [478767548](https://github.com/Azure/azure-iot-sdk-python/pull/649#discussion_r478767548) [](ancestors = 478767548)